pkg/client/clientset/versioned/clientset.go

```diff
@@ -57,6 +62,17 @@ func (c *Clientset) Ceph() cephv1alpha1.CephV1alpha1Interface {
 	return c.cephV1alpha1
 }
 
+// CockroachdbV1alpha1 retrieves the CockroachdbV1alpha1Client
+func (c *Clientset) CockroachdbV1alpha1() cockroachdbv1alpha1.CockroachdbV1alpha1Interface {
+	return c.cockroachdbV1alpha1
+}
+
+// Deprecated: Cockroachdb retrieves the default version of CockroachdbClient.
+// Please explicitly pick a version.
+func (c *Clientset) Cockroachdb() cockroachdbv1alpha1.CockroachdbV1alpha1Interface {
```

Comment:
See above. If you want people to use the one with a version, don't give anyone a choice to use an unversioned function.

Reply:
Same, this is purely generated code.